packages/frontend/src/utils/getTranslationsFromMultisigRequest.js

```diff
@@ -0,0 +1,76 @@
+import { formatNearAmount } from 'near-api-js/lib/utils/format';
+
+const NEAR_FRACTIONAL_DIGITS = 4;
+
+const formatNear = (amount) => formatNearAmount(amount, NEAR_FRACTIONAL_DIGITS);
```

Comment:
I think we can re-use the existing NEAR formatting function in the balance [helper](https://github.com/near/near-wallet/blob/master/packages/frontend/src/components/common/balance/helpers.js) file too

Reply:
Sounds good 👍🏽